### DISCUSSION TITLE
Where can I download an older version?

### DISCUSSION THREAD

**Discussion by tymime** [The-Wineskin-Project/WineskinServer#228] [Q&A] [answered]
I tried updating Winery to the latest version (hoping that an error I was encountering would be fixed), but realized too late that all wrappers it creates only run on High Sierra and above (I'm stuck on Sierra for a number of reasons, don't ask). Is there a way to downgrade?

**Comment by Gcenx** [comment, marked as answer]
You can find the older master wrappers in the git history however I won’t provide any support for this configuration, after the recent cert expiration it’s frankly pushing it supporting High Sierra & Mojave as is most cases the expired certs need to be updated manually.

**Reply by eierfrucht** [reply]
@gcenx forgive my ignorance but how do I obtain an older master wrapper from the git history and more importantly how do I tell which is the last master wrapper to support High Sierra and Mojave? Could you perhaps post a link to it in the project description? Looks like people have posted questions of this kind for three or four times already.

I'm trying to run something on a machine stuck at High Sierra and the program in question has always worked for me with really, really old wrappers and Wine/Crossover engines...
